test/test.ts

```diff
@@ -151,3 +160,16 @@ function mockExists() {
       .get('/v1/projects/el-gato/locations/us-central1/functions/%F0%9F%A6%84')
       .reply(200);
 }
+
+/**
+ * @see https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions/call
+ */
+function mockCall() {
+  return nock('https://cloudfunctions.googleapis.com')
+      .post('/v1/*')
```

Comment:
this should be `.post('/v1/%F0%9F%A6%84:call')`

Reply:
Fixed. Thanks.